src/utils.py

```diff
@@ -86,8 +88,8 @@ class ServerHTTPSConnection(http.client.HTTPSConnection):
     def __init__(self, config, server, port, cert_file):
         self.no_ssl = config.suppress_ssl or not FEAT_SSL
         if self.no_ssl:
-            if config.use_proxy == True:
-                http.client.HTTPSConnection.__init__(self, config.proxy_url, config.proxy_port, context=context)
```

Comment:
@jsouthern-r7 I wondered the same thing. 
But in this case the if block is no_ssl and context wasn't defined in this case. 
Which is why I removed context.